doc/sphinx/source/project/about.rst

```diff
@@ -1,108 +1,224 @@
 .. about: 
 
-Overview
-========
 
+############################
+FAQs, Features, and Overview
+############################
 
-Welcome to the documentation site for |InG|, an open-source C++ Compiler, 
-Library, and runtime suite for Deep Learning frameworks running training and inference on 
-:abbr:`Deep Neural Network (DNN)` models. nGraph is framework-neutral and can be 
-targeted for programming and deploying :abbr:`Deep Learning (DL)` applications 
-on the most modern compute and edge devices.   
+* :ref:`comparisons`
+* :ref:`faq`
+* :ref:`features`
+* :ref:`whats_next`
 
-Features
---------
 
-:ref:`no-lockin`
-:ref:`framework-flexibility`
+.. _comparisons: 
 
+Comparisons  
+===========
 
-.. _no-lockin:
+How does nGraph compare to similar efforts in the industry?
+------------------------------------------------------------
 
-Develop without lock-in
-~~~~~~~~~~~~~~~~~~~~~~~
+Additional and specialized work is being done cross-industry. With nGraph, 
+work done for other similar efforts can be ported and tested natively. For 
+example, many high-performance kernels built with Intel MKL-DNN can be 
+run with nGraph providing another layer for creating specialized sub-graphs
+that themselves can be further optimized when specialized hardware is 
+available.  
+
+.. TODO:  Add link to subgraph_snippets docs when PR  
+
+.. important:: Our design philosophy is that the graph is not a script for 
+   running kernels; rather, our compilation will match ``ops`` to appropriate 
+   kernels for available backends. Thus, we expect that adding of new Core 
+   ops should be infrequent and that most functionality instead gets added 
+   with new functions that build sub-graphs from existing core ops.
+
+
+* The latest **PlaidML Backend** adds an extra layer of utility to an 
```

Comment:
I'm not sure what this means.

Reply:
This section on "comparisons" was a hand-down request for us to compare nGraph to other "graph compilers" ... which I am not sure makes sense, since there's not much in the way of  graph compilers that actually work.  Since PlaidML is a "tensor compiler", I am assuming you can talk here about the specialized capabilities of PlaidML as a compiler when it's called as part of nGraph.

Reply:
Let's clean this up as a follow-up.

Reply:
Took this section out for now